python/foglamp/services/core/server.py

```diff
@@ -672,8 +747,6 @@ def main():
         if command == 'start':
             Server().start()
         elif command == 'stop':
-            Server().stop_storage()
-            Server().stop_service_monitor()
-            # scheduler has signal binding
+            Server().stop()
```

Comment:
I will remove `def main():` and possibly the entry point from setup.py as well ; as we are not using this; instead we are calling everything from shell scripts with `python3 -m ....`